app/bundles/PageBundle/EventListener/DetermineWinnerSubscriber.php

```diff
@@ -58,7 +60,8 @@ public static function getSubscribedEvents()
     public function onDetermineBounceRateWinner(DetermineWinnerEvent $event)
     {
         //find the hits that did not go any further
-        $repo      = $this->em->getRepository('MauticPageBundle:Hit');
+        /** @var HitRepository $repo */
+        $repo      = $this->em->getRepository(Hit::class);
```

Comment:
Can we inject the `HitRepository` instead of the entity manager? The em isn't used for any reason in the subscriber other than to get the repo

Reply:
https://github.com/mautic/mautic/pull/8148/commits/fe7e997d8336d87fb176eb5e33c0f5e315361fdb